.github/workflows/tests_linux_x86.yml

```diff
@@ -259,7 +259,7 @@ jobs:
           python -m pip install pip~=22.0
           python -m pip install ninja cmake custatevec-cu11 pytest pytest-mock flaky pytest-cov
           # Sync with latest master branches
-          python -m pip install --index-url https://test.pypi.org/simple/ pennylane-lightning --pre --force-reinstall --no-deps
+          python -m pip install git+https://github.com/PennyLaneAI/pennylane-lightning.git@master --force-reinstall --no-deps
```

Comment:
Let's accept the build here for now, but we can update this to use the TestPyPI wheels after release, as it skips the build process.